src/ratings/Ratings.js

```diff
@@ -9,15 +9,35 @@ import axios from 'axios';
 
 const Ratings = ({product}) => {
   const [metaData, setMetaData] = useState({});
-  const [reviewData, setReviewData] = useState({});
-
   const [reviewSort, setReviewSort] = useState('');
   const [reviewCount, setCountReview] = useState(2);
   const [productId , setProductId] = useState(19089);
-  const [starSort, setStarSort] = useState(null);
+
+  //State to manage star sorting
+  const [reviewData, setReviewData] = useState({});
+  const [postSort, setPostSort] = useState([]);
+  const [starSort, setStarSort] = useState({
+    '1': false,
+    '2': false,
+    '3': false,
+    '4': false,
+    '5': false
+  });
+
+  console.log('review data: ', reviewData);
```

Comment:
Hi @ Sam mind commenting the console.logs() out? I should probably check mine...

Reply:
Oh yeah good idea! I thought I already had. Forgot to.

Reply:
Haha, nice! Looks good! I will fix conflict and merge!

Reply:
Ok yeah just removed them! Thanks!